app/src/main/java/org/fossasia/openevent/Adapters/SpeakersListAdapter.java

```diff
@@ -0,0 +1,83 @@
+package org.fossasia.openevent.Adapters;
+
+import android.content.Context;
+import android.net.Uri;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.data.Speaker;
+
+import java.util.List;
+
+/**
+ * Created by MananWason on 11-06-2015.
+ */
+public class SpeakersListAdapter extends RecyclerView.Adapter<SpeakersListAdapter.ViewHolder> {
+    List<Speaker> speakers;
+    private Context context;
```

Comment:
I haven't tried getting the context from the holder. I'll do it now and get back to you.